backend/config/corpora-api.yml

```diff
@@ -486,10 +491,16 @@ paths:
               schema:
                 type: object
                 properties:
-                  dataset_uuid:
+                  dataset_id:
                     type: string
```

Comment:
Should this just be `$ref: "#/components/schemas/dataset_uuid"`

Reply:
Will fix